src/main.py

```diff
@@ -57,6 +64,8 @@ class Settings(BaseSettings):
     shortcut_api_token: str = Field(..., env="INPUT_SHORTCUT_API_TOKEN")
     open_id: str = Field(..., env="INPUT_PR_OPENED")
     closed_id: str = Field(..., env="INPUT_PR_CLOSED")
+    wip_id: str = Field(env="INPUT_PR_WIP", default="")
+    wip_keyword: str = Field(env="INPUT_WIP_KEYWORD", default="")
```

Comment:
we need to add these field to `action.yaml` and README.md

Reply:
I updated that part!

Reply:
Thanks!